<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">background-color: #F4F1DE;
border-radius: 20px;</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="lesson_page">
       <widget class="QFrame" name="top_frame">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>1280</width>
          <height>80</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>1280</width>
          <height>80</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>80</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">border: none;
background-color: #E07A5F;
border-top-left-radius: 20px;
border-top-right-radius: 20px;
color: #3D405B; </string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="btn_back_to_home">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>60</height>
            </size>
           </property>
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>4</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton {
	background-color: ;
	border: none;
}

QPushButton:hover:!pressed
{
	background-color: #F4F1DE;
	border-radius: 30px;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset>
             <normaloff>../Images/back.png</normaloff>../Images/back.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
           <property name="autoDefault">
            <bool>false</bool>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="lbl_headline">
           <property name="font">
            <font>
             <family>Hind Siliguri Medium</family>
             <pointsize>27</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:#3D405B;
padding-left: 5px;</string>
           </property>
           <property name="text">
            <string>পাঠসূচি</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QFrame" name="bottom_frame">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>1281</width>
          <height>771</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>1280</width>
          <height>770</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QTextEdit" name="textEdit">
         <property name="geometry">
          <rect>
           <x>310</x>
           <y>220</y>
           <width>631</width>
           <height>87</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>25</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">border: 3px solid rgb(224, 122, 95);
font: 25pt &quot;MS Shell Dlg 2&quot;;
padding-left: 10px;</string>
         </property>
         <property name="placeholderText">
          <string>একটি ফোল্ডার নির্বাচন করুন</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton_3">
         <property name="geometry">
          <rect>
           <x>460</x>
           <y>350</y>
           <width>350</width>
           <height>71</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>350</width>
           <height>350</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
border: 5px solid #E07A5F;
}

QPushButton:hover:!pressed {
background-color: #3D405B;
color: #F4F1DE;
}</string>
         </property>
         <property name="text">
          <string>নির্বাচন করুন</string>
         </property>
         <property name="iconSize">
          <size>
           <width>400</width>
           <height>400</height>
          </size>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="home_page">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>350</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
border: 5px solid #E07A5F;
}

QPushButton:hover:!pressed {
background-color: #3D405B;
color: #F4F1DE;
}

QPushButton:pressed {
background-color: #F4F1DE;
color: #E07A5F;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Images/lessons_icon.png</normaloff>../Images/lessons_icon.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>400</width>
            <height>400</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="maximumSize">
           <size>
            <width>350</width>
            <height>350</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
border: 5px solid #E07A5F;
}

QPushButton:hover:!pressed {
background-color: #3D405B;
color: #F4F1DE;
}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../Images/evaluation_icon.png</normaloff>../Images/evaluation_icon.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>400</width>
            <height>400</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
